README.md

```diff
@@ -91,7 +91,7 @@ No software from this repository is required.
     scratches, gradients, and JPEG artifacts.
 
 2. Process the identity image with a target software you prefer.
-    Speaking of Instagram, you need to transfer the identity
+    Speaking of mobile apps and their filters, you need to transfer the identity
```

Comment:
It's obvious about Instagram ) Otherwise just "process with software" would be enough.

Reply:
We can go with just `apps and their filters`, or `say, if you were using Instagram, you'd have to`.